api/config/config_local.json

```diff
@@ -46,9 +46,9 @@
     "operationalTerraWorkspaceName": "aouwgscohortextraction",
     "operationalTerraWorkspaceBucket": "fc-56d2f6f5-3efa-46f7-8c01-0911fd77f888",
     "extractionMethodConfigurationNamespace": "aouwgscohortextraction-test",
-    "extractionMethodConfigurationName": "GvsExtractCohortFromSampleNames",
-    "extractionMethodConfigurationVersion": 15,
-    "extractionMethodLogicalVersion": 3,
+    "extractionMethodConfigurationName": "joelproto",
+    "extractionMethodConfigurationVersion": 1,
+    "extractionMethodLogicalVersion": 4,
```

Comment:
Think of this as the Workflow Version, from the RWB perspective.  We have the ability to use this as a discriminator, if we need different Methods for CDRv7 vs CDRv8.